readme.md

```diff
@@ -156,25 +153,25 @@ set the proxy_pass rule to that machine.
 
 * key [not set by default]
 
-  A secret. If set any system that wants to access this prerender setup,
-  _must_ provide an Authorization: Bearer <key> header. Obviously this
-  is only safe if used over http*s*.
+	A secret. If set any system that wants to access this prerender setup,
```

Comment:
I liked that the paragraphs were indented at the same level as the first word... :'(